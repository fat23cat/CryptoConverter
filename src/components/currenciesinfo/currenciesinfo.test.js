import React from 'react';
import ReactDOM from 'react-dom';
import CurrenciesInfo from './currenciesinfo.js';

describe('components', () => {
  it('should render a CurrenciesInfo', () => {
   const wrapper = (<CurrenciesInfo />);
   expect(wrapper).toMatchSnapshot();
  })
})
