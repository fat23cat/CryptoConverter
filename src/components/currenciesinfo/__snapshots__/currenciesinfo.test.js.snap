// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components should render a CurrenciesInfo 1`] = `<Connect(CurrenciesInfo) />`;
